test/e2e/specs/plugins/plugins__search.ts

```diff
@@ -63,4 +64,21 @@ describe( DataHelper.createSuiteTitle( 'Plugins search' ), function () {
 			await pluginsPage.validateExpectedSearchResultFound( 'Jetpack Protect' );
 		}
 	} );
+
+	it( 'Navigate back to the default plugins page when searching from categories pages', async function () {
+		window.location = {
+			...window.location,
+			href: 'https://calypso.localhost:3000/plugins',
```

Comment:
**question:blocking** - would this work in environments other than the locally run webapp?

Reply:
Removed.